schema/me/conversation/index.js

```diff
@@ -190,7 +190,18 @@ export const ConversationFields = {
   last_message_open: {
     type: GraphQLString,
     description: "Timestamp if the user opened the last message, null in all other cases",
-    resolve: conversation => null, // eslint-disable-line no-unused-vars
+    resolve: (conversation, options, request, { rootValue: { accessToken } }) => {
+      const radiationMessageId = get(conversation, "_embedded.last_message.radiation_message_id")
+      const impulseParams = { conversation_id: conversation.id, radiationMessageId, "expand[]": "deliveries" }
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
+        return impulse.with(data.token, { method: "GET" })(`message_details`, impulseParams).then(radiationMessage => {
+          const relevantDelivery = radiationMessage.deliveries.find(d => d.email === conversation.from_email)
```

Comment:
Basically we fetch the details/deliveries for the last message (embedded in the conversation).

We find the user's delivery, and return that `opened_at`.